From 0f4ac568f5d63e9d57b0c10c8f5620df1ae3aa94 Mon Sep 17 00:00:00 2001
From: Aparna M <a-m1@ti.com>
Date: Fri, 13 May 2022 18:13:09 +0530
Subject: [PATCH] create-sdcard: Copy wificfg file to boot partition for am62xx

WiFi demo supported in am62xx SK requires a wificfg file to be copied to
the boot partition of the SD card. Add support in the script to copy the
same.

Signed-off-by: Aparna M <a-m1@ti.com>
---
 create-sdcard.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/create-sdcard.sh b/create-sdcard.sh
index fa7dadb..4af6c94 100644
--- a/create-sdcard.sh
+++ b/create-sdcard.sh
@@ -1129,7 +1129,7 @@ if [ $BOOTPATHOPTION -eq 1 ] ; then
 
 	echo ""
 
-	if [ "$TARIMAGETYPE" == "default" ] && [ "$TARDEVICENAME" == "am64xx" ] ; then
+	if [ "$TARIMAGETYPE" == "default" ] && ( [ "$TARDEVICENAME" == "am64xx" ] || [ "$TARDEVICENAME" == "am62xx" ] ); then
 	       	if [ "$WIFICFG" != "" ] ; then
 			cp $BOOTFILEPATH/$WIFICFG $PATH_TO_SDBOOT/wificfg
 	                echo "wificfg copied"
-- 
2.17.1

